packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -543,15 +552,16 @@ contract OPContractsManager is ISemver {
                     );
                 }
 
+                // TODO: add new prestate
```

Comment:
Is this still a TODO?

Reply:
Also I'd suggest adding a comment like this
```suggestion
                // Deploy and set a new permissioned game to update its prestate
```

Reply:
No longer a todo. Accepting this change.